dev-tools/reconfigurator-cli/src/main.rs

```diff
@@ -314,6 +346,29 @@ struct BlueprintPlanArgs {
     collection_id: Uuid,
 }
 
+#[derive(Debug, Args)]
+struct BlueprintEditArgs {
+    /// id of the blueprint to edit
+    blueprint_id: Uuid,
+    /// "creator" field for the new blueprint
+    #[arg(long)]
+    creator: Option<String>,
+    /// "comment" field for the new blueprint
+    #[arg(long)]
+    comment: Option<String>,
+    #[command(subcommand)]
+    edit_command: BlueprintEditCommands,
+}
+
+#[derive(Debug, Subcommand)]
+enum BlueprintEditCommands {
+    /// add a Nexus instance to a particular sled
+    AddNexus {
+        /// sled on which to deploy the new instance
+        sled_id: Uuid,
```

Comment:
Gosh I really want to start using `TypedUuid` for this soon

Reply:
We have some documentation at https://github.com/oxidecomputer/omicron/tree/main/uuid-kinds, and you can see that directory's history. But let me do another example and write up a more detailed guide.